documentation/website/src/handbook/integration/serenity-2-migration-guide.md

```diff
@@ -0,0 +1,688 @@
+   * [Serenity/JS 1.x implementation of Hide Task](/handbook/tutorials/serenity-2-migration-guide.html#serenity-js-1-x-implementation-of-hide-task)
+   * [Serenity/JS 2.x implementation of Hide Task](/handbook/tutorials/serenity-2-migration-guide.html#serenity-js-2-0-implementation-of-hide-task)
+
+## Building and viewing documentation locally
+
+Most Serenity/JS classes have been comprehensively documented using [JSDoc](https://devdocs.io/jsdoc/about-getting-started#getting-started) along with usage examples, so this is a very useful documentation source that you can use.
+Now the [2.x website](https://serenity-js.org/) is publicly available, [module documentation](https://serenity-js.org/modules) can be viewed online.
+But just in case you want to build a branch of the documenation locally, you can follow these steps:
+
+* From your terminal window, clone the repository: `git clone https://github.com/jan-molak/serenity-js.git`
+* Move into the project directory: `cd serenity-js`
+* Update dependencies: `npm install`
+* Build the documentation: `make clean verify report site`
+* Move into the build documentation directory: `cd documentation/website`
+* Serve the documentation: `npm start`
+* Open the documentation in your browser at: `http://localhost:3000`
+* For instance, you'll find this guide [here](http://localhost:3000/handbook/tutorials/serenity-2-migration-guide.html), nestled in the [tutorials](http://localhost:3000/handbook/tutorials/index.html)
+
+## Migrating package.json
+### Updating dependencies
+Various libraries are no longer needed. Here's a high-level summary of changes:
+ * Chai is now transitively brought in via `@serenity-js/assertions` 
+ * mocha is not yet supported so if you used that in 1.x, you'll need to switch to jasmine. 
+ * serenity-cli has been replaced by `@serenity-js/serenity-bdd`. 
+
+So remove all of these from your dependencies section:
+   ```javascript
+    "@types/chai"
+    "@types/chai-as-promised"
+    "@types/mocha"
+    "chai"
+    "chai-as-promised"
+    "chai-smoothie"
+    "mocha",
+    "serenity-cli"
+```    
+Now v2.x is properly published, versions can be set to "latest" if you want to get the latest. 
+However, if you want to want to stay on a fixed version, use that instead.
+
+You can find out the latest available version by visiting the [releases page](https://github.com/jan-molak/serenity-js/releases). :
+   ```javascript
+    "@serenity-js/assertions": "latest",
+    "@serenity-js/console-reporter": "latest",
+    "@serenity-js/core": "latest",
+    "@serenity-js/cucumber": "latest",
+    "@serenity-js/jasmine": "latest",
+    "@serenity-js/protractor": "latest",
+    "@serenity-js/rest": "latest",
+    "@serenity-js/serenity-bdd": "latest",
+    "@types/express": "4.16.1",
+    "@types/jasmine": "3.4.0",
+    "@types/node": "10.14.20",
+    "@types/sinon": "4.3.3",
+    "@types/sinon-chai": "3.2.2",
```

Comment:
I wonder if we need to mention the non-Serenity/JS-specific deps like Sinon, Node or even TypeScript?
Wouldn't this confuse people?

How about we tell them to use the `@serenity-js/*` packages at `^2.0.0` and add other dependencies they might need tor their type of project, based on the template repositories?